### DISCUSSION TITLE
PHP-SOAP Module

### DISCUSSION THREAD

**Discussion by jrmeireles** [roundcube/roundcubemail-docker#246] [Ideas]
Why is the php-soap module not part of roundcube docker?

**Comment by thomascube** [comment]
And why should it be? Roundcube doesn't use anything SOAP.

**Comment by jrmeireles** [comment]
Typically, integration with other applications like ispconfig uses SOAP.

---
Att.
Jackson R. Meireles
***@***.***
linkedin.com/in/jrmeireles

Em 16/05/2024 18:42, Thomas B. escreveu:

> And why should it be? Roundcube doesn't use anything SOAP.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/roundcube/roundcubemail-docker/discussions/246#discussioncomment-9463067
[2]

**Comment by thomascube** [comment]
This is very specific and for such cases we suggest to build your own Docker image with the additions (and Roundcube plugins) you need. We can't pack everything into the default image.
